File: Programming-Basics-With-JavaScript/Homeworks-And-Labs/06.Nested-Loop/03-additional/04.carNumber.js
function carNumber(input) {

    let start = Number(input[0]);
    let end = Number(input[1]);
    let printLine = '';

    for (let i = start; i <= end; i++) {
        for (let j = start; j <= end; j++) {
            for (let k = start; k <= end; k++) {
                for (let l = start; l <= end; l++) {
                    if (i > l && (j + k) % 2 == 0 && ((i % 2 == 0 && l % 2 != 0) || (i % 2 != 0 && l % 2 == 0))) {
                        printLine += `${i}${j}${k}${l} `;
                    }
                }
            }
        }
    }

    console.log(printLine);
}
carNumber(['5', '8']);

File: Python_Basics/Homeworks-And-Labs/06.nested_loops/03.additional/04.car_number.py
start = int(input())
end = int(input())
print_line = ''

for i in range(start, end + 1):
    for j in range(start, end + 1):
        for k in range(start, end + 1):
            for l in range(start, end + 1):
                if i > l and (j + k) % 2 == 0 and ((i % 2 == 0 and l % 2 != 0) or (i % 2 != 0 and l % 2 == 0)):
                    print_line += f'{i}{j}{k}{l} '

print(print_line)
